.github/ISSUE_TEMPLATE/format_request.yml

```diff
@@ -0,0 +1,56 @@
+name: Request Format Support
+description: Request support for a new format.
+title: "[New Format]: "
+labels: ["enhancement", "data interfaces"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Thank you for your suggestion!
+
+        We welcome any ideas about how to make **NeuroConv** better for the community.
+
+        Please keep in mind that adding support for new formats may take some time.
+  - type: textarea
+    id: what-format
+    attributes:
+      label: What format would you like to see added to NeuroConv?
+      description: What acquisition hardware or software is responsible for producing the output files?
+    validations:
+      required: true
+  - type: textarea
+    id: format-documentation
+    attributes:
+      label: Does the format have any documentation?
+      description: A link or other details to rich documentation describing the folder/file structure.
+  - type: textarea
+    id: example-files
+    attributes:
+      label: Do you have any example files you are willing to share?
+      description: We require small example files for our testing suite to ensure we properly support the format.
+    validations:
+      required: true
```

Comment:
```suggestion
```

We do want files but I don't want to make this a requirement of requesting a format interface. We should be able to file a record without an example file on hand.